src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag

```diff
@@ -132,127 +134,117 @@
                                        <c:if test="${frc.showGiverNameToResponseGiver}">checked="checked"</c:if>>
                             </td>
                         </tr>
-                        <c:if test="${frc.responseVisibleToRecipient}">
```

Comment:
Ok I'll take a look at them tomorrow, I wasn't sure what they were used for and I didn't want to instantiate variables that wouldn't be used elsewhere, I'll take a look first thing tomorrow and make the appropriate changes, thanks for pointing me to the right direction!